examples/tutorials/Part 02 - Intro to Federated Learning.ipynb

```diff
@@ -113,9 +140,21 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 4,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "ename": "ModuleNotFoundError",
+     "evalue": "No module named 'syft'",
+     "output_type": "error",
+     "traceback": [
+      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
```

Comment:
Q: This is ok?

Reply:
this was not changed by me,  actually, i only changed two files, probably some error while I rebased my PySyft repo, I changed serde.py and grid.clients.datacentric files I will update this.